### DISCUSSION TITLE
5.15.34无线需要修复

### DISCUSSION THREAD

**Discussion by yaya131** [Boos4721/openwrt#116] [Bugs/问题反馈]
Sun Apr 17 10:05:20 2022 daemon.notice hostapd: Configuration file: /var/run/hostapd-phy0.conf (phy wlan0) --> new PHY
Sun Apr 17 10:05:20 2022 daemon.err hostapd: Could not set interface wlan0 flags (UP): Cannot send after socket shutdown
Sun Apr 17 10:05:20 2022 daemon.err hostapd: nl80211: Could not set interface 'wlan0' UP
Sun Apr 17 10:05:20 2022 daemon.notice hostapd: nl80211: deinit ifname=wlan0 disabled_11b_rates=0
Sun Apr 17 10:05:20 2022 daemon.err hostapd: nl80211 driver initialization failed.
Sun Apr 17 10:05:20 2022 daemon.notice hostapd: wlan0: CTRL-EVENT-TERMINATING
Sun Apr 17 10:05:20 2022 daemon.err hostapd: hostapd_free_hapd_data: Interface wlan0 wasn't started
Sun Apr 17 10:05:20 2022 daemon.notice netifd: radio0 (6841): Command failed: Invalid argument
Sun Apr 17 10:05:20 2022 kern.err kernel: [  447.441002] ath11k c000000.wifi: failed to enable PMF QOS: (-108
Sun Apr 17 10:05:20 2022 daemon.notice netifd: radio0 (6841): Device setup failed: HOSTAPD_START_FAILED
Sun Apr 17 10:05:20 2022 daemon.notice netifd: Wireless device 'radio0' set retry=0
Sun Apr 17 10:05:20 2022 daemon.crit netifd: Wireless device 'radio0' setup failed, retry=0
Sun Apr 17 10:05:20 2022 daemon.notice netifd: Wireless device 'radio0' is now down

Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.688028] ath11k c000000.wifi: failed to send WMI_PDEV_BSS_CHAN_INFO_REQUEST cmd
Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.691719] ath11k c000000.wifi: failed to send pdev bss chan info request
Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.699367] ath11k c000000.wifi: failed to send WMI_PDEV_SET_PARAM cmd
Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.705953] ath11k c000000.wifi: Failed to set beacon mode for VDEV: 0
Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.712509] ath11k c000000.wifi: failed to send WMI_BCN_TMPL_CMDID
Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.718988] ath11k c000000.wifi: failed to submit beacon template command: -108
Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.725137] ath11k c000000.wifi: failed to update bcn template: -108
Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.732367] ath11k c000000.wifi: failed to send WMI_VDEV_SET_PARAM_CMDID
Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.738971] ath11k c000000.wifi: Failed to set dtim period for VDEV 0: -108
Sun Apr 17 10:00:39 2022 kern.warn kernel: [  165.745745] ath11k c000000.wifi: failed to send WMI_VDEV_SET_PARAM_CMDID

**Comment by Boos4721** [comment]
NEED ATH11K FIRWARE INFO

**Reply by googlewmb** [reply]
> 需要 ATH11K 信息

ipq807x所机器如此。另外集成sfe通用加速。在特定的场景如科学上网，手机连无线科学上网。路由会出现内存骤降至0并死机。具体内核及日志均没有该情况的详细的日志。

**Reply by yaya131** [reply]
> > 需要 ATH11K 信息
> 
> ipq807x所机器如此。另外集成sfe通用加速。在特定的场景如科学上网，手机连无线科学上网。路由会出现内存骤降至0并死机。具体内核及日志均没有该情况的详细的日志。

的确存在。AX6及AX3600&AX9000均是如此。

**Reply by googlewmb** [reply]
5.15.34暂不支持nss相关加速。

**Reply by Boos4721** [reply]
> 5.15.34暂不支持nss相关加速。

NA, WORKS

**Reply by googlewmb** [reply]
> > 5.15.34暂不支持nss相关加速。
> 
> 不，工作

是的

**Reply by yaya131** [reply]
> > 5.15.34暂不支持nss相关加速。
> 
> NA, WORKS

  387.779851]  el0t_64_sync_handler+0xa4/0x130
[  387.782891]  el0t_64_sync+0x184/0x188
[  387.787402] Mem-Info:
[  387.790880] active_anon:1088 inactive_anon:1340 isolated_anon:0
[  387.790880]  active_file:1564 inactive_file:675 isolated_file:0
[  387.790880]  unevictable:0 dirty:98 writeback:0
[  387.790880]  slab_reclaimable:2556 slab_unreclaimable:53801
[  387.790880]  mapped:2006 shmem:214 pagetables:225 bounce:0
[  387.790880]  kernel_misc_reclaimable:0
[  387.790880]  free:6337 free_pcp:424 free_cma:0
[  387.806560] Node 0 active_anon:4884kB inactive_anon:5360kB active_file:4524kB inactive_file:3600kB unevictable:0kB isolated(anon):0kB isolated(file):92kB mapped:6960kB dirty:392kB writeback:0kB shmem:856kB writeback_tmp:0kB kernel_stack:2072kB pagetables:900kB all_unreclaimable? no
[  387.831302] DMA free:31664kB min:24576kB low:28672kB high:32768kB reserved_highatomic:4096KB active_anon:4536kB inactive_anon:6148kB active_file:2844kB inactive_file:2808kB unevictable:0kB writepending:392kB present:524288kB managed:417372kB mlocked:0kB bounce:0kB free_pcp:896kB local_pcp:488kB free_cma:0kB
[  387.858636] lowmem_reserve[]: 0 0 0 0
[  387.880885] DMA: 2716*4kB (UMEH) 1295*8kB (UMEH) 535*16kB (UMEH) 87*32kB (UMEH) 14*64kB (UME) 3*128kB (M) 2*256kB (UM) 0*512kB 0*1024kB 0*2048kB 0*4096kB = 34360kB
[  387.884393] 1067 total pagecache pages
[  387.898828] 0 pages in swap cache
[  387.902595] Swap cache stats: add 4076, delete 4076, find 0/517
[  387.905983] Free swap  = 192508kB
[  387.911778] Total swap = 207868kB
[  387.915183] 131072 pages RAM
[  387.918572] 0 pages HighMem/MovableOnly
[  387.921433] 26729 pages reserved
[  387.924991] Tasks state (memory values in pages):
[  387.928497] [  pid  ]   uid  tgid total_vm      rss pgtables_bytes swapents oom_score_adj name
[  387.933106] [    705]    81   705      308       37    32768       28             0 ubusd
[  387.941616] [    706]     0   706      209       17    32768       11             0 askfirst
[  387.949858] [   1760]   514  1760      302       60    32768       18             0 logd
[  387.958364] [   1808]     0  1808      396       52    32768       29             0 rpcd
[  387.966408] [   2223]     0  2223     1568       91    45056       83             0 hostapd
[  387.974509] [   2224]     0  2224     1539        0    45056       85             0 wpa_supplicant
[  387.982605] [   2281]     0  2281      459       76    32768       35             0 netifd
[  387.991623] [   2336]     0  2336      360       52    28672       30             0 odhcpd
[  387.999855] [   2553]     0  2553     1352       49    40960      148             0 uhttpd
[  388.008094] [   2928]     0  2928      310        0    28672       12             0 udhcpc
[  388.016322] [   2937]     0  2937      248       21    28672       15             0 odhcp6c
[  388.024612] [   3344]     0  3344      259       34    28672       14             0 dropbear
[  388.032849] [   3346]     0  3346      311        0    32768       13             0 ntpd
[  388.041531] [   4194]     0  4194      212        0    28672       13             0 ntpclient
[  388.049602] [   5658]     0  5658     1489       55    40960       86             0 ttyd
[  388.058024] [  10601]     0 10601      310       34    28672       11             0 crond
[  388.066157] [  11316] 65534 11316   178672     1869   102400     2696             0 clash
[  388.074241] [  11749]   453 11749      365       94    32768       13             0 dnsmasq
[  388.082425] [  11796]     0 11796      312       33    28672       14             0 openclash_watch
[  388.090554] [  11822]     0 11822      310        0    32768       11             0 sleep
[  388.099608] [  12275]     0 12275     1019      118    36864      356             0 luci
[  388.107912] [  12308]     0 12308      313      194    28672        0             0 ethinfo
[  388.116062] [  12408]     0 12408      313       17    28672        0             0 ethinfo
[  388.124167] [  12410]     0 12410      314       18    28672        0             0 ethinfo
[  388.132490] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/,task=clash,pid=11316,uid=65534
[  388.140892] Out of memory: Killed process 11316 (clash) total-vm:714688kB, anon-rss:7336kB, file-rss:140kB, shmem-rss:0kB, UID:65534 pgtables:100kB oom_score_adj:0
[  388.161529] oom_reaper: reaped process 11316 (clash), now anon-rss:0kB, file-rss:12kB, shmem-rss:0kB
[  472.631244] clash invoked oom-killer: gfp_mask=0xcc0(GFP_KERNEL), order=0, oom_score_adj=0
[  472.631310] CPU: 3 PID: 13150 Comm: clash Not tainted 5.15.34 #0
[  472.638424] Hardware name: Redmi AX6 (DT)
[  472.644584] Call trace:
[  472.648484]  dump_backtrace+0x0/0x160
[  472.650746]  show_stack+0x18/0x30
[  472.654564]  dump_stack_lvl+0x7c/0xa0
[  472.657862]  dump_stack+0x18/0x34
[  472.661507]  dump_header+0x48/0x184
[  472.664807]  oom_kill_process+0x1e0/0x1ec
[  472.668106]  out_of_memory+0x1cc/0x320
[  472.672273]  __alloc_pages_slowpath.constprop.0+0x73c/0xa34
[  472.675922]  __alloc_pages+0x220/0x24c
[  472.681387]  skb_page_frag_refill+0x10c/0x14c
[  472.685207]  sk_page_frag_refill+0x20/0x90
[  472.689634]  tcp_sendmsg_locked+0x398/0xbe0
[  472.693627]  tcp_sendmsg+0x38/0x60
[  472.697715]  inet6_sendmsg+0x44/0x70
[  472.701179]  sock_write_iter+0x98/0xf0
[  472.704911]  new_sync_write+0x148/0x154
[  472.708470]  vfs_write+0x214/0x2c0
[  472.712201]  ksys_write+0xc8/0xe0
[  472.715674]  __arm64_sys_write+0x1c/0x30
[  472.719060]  invoke_syscall.constprop.0+0x5c/0x104
[  472.723057]  do_el0_svc+0x74/0x16c
[  472.727654]  el0_svc+0x18/0x54
[  472.731038]  el0t_64_sync_handler+0xa4/0x130
[  472.734079]  el0t_64_sync+0x184/0x188
[  472.738636] Mem-Info:
[  472.742066] active_anon:161 inactive_anon:238 isolated_anon:0
[  472.742066]  active_file:32 inactive_file:14 isolated_file:0
[  472.742066]  unevictable:0 dirty:0 writeback:0
[  472.742066]  slab_reclaimable:1967 slab_unreclaimable:62585
[  472.742066]  mapped:68 shmem:5 pagetables:186 bounce:0
[  472.742066]  kernel_misc_reclaimable:0
[  472.742066]  free:5781 free_pcp:415 free_cma:0
[  472.756664] Node 0 active_anon:40kB inactive_anon:440kB active_file:128kB inactive_file:56kB unevictable:0kB isolated(anon):0kB isolated(file):0kB mapped:272kB dirty:0kB writeback:0kB shmem:20kB writeback_tmp:0kB kernel_stack:2016kB pagetables:744kB all_unreclaimable? no
[  472.780532] DMA free:18416kB min:16384kB low:20480kB high:24576kB reserved_highatomic:4096KB active_anon:344kB inactive_anon:528kB active_file:108kB inactive_file:0kB unevictable:0kB writepending:0kB present:524288kB managed:417372kB mlocked:0kB bounce:0kB free_pcp:2244kB local_pcp:388kB free_cma:0kB
[  472.807265] lowmem_reserve[]: 0 0 0 0
[  472.829495] DMA: 34*4kB (EH) 519*8kB (UMEH) 523*16kB (UH) 14*32kB (H) 2*64kB (H) 4*128kB (H) 1*256kB (H) 0*512kB 0*1024kB 0*2048kB 0*4096kB = 14000kB
[  472.833142] 36 total pagecache pages
[  472.846511] 8 pages in swap cache
[  472.850135] Swap cache stats: add 9791, delete 9790, find 32/1398
[  472.853353] Free swap  = 183292kB
[  472.859519] Total swap = 207868kB
[  472.862722] 131072 pages RAM
[  472.866021] 0 pages HighMem/MovableOnly
[  472.868987] 26729 pages reserved
[  472.872532] Tasks state (memory values in pages):
[  472.876006] [  pid  ]   uid  tgid total_vm      rss pgtables_bytes swapents oom_score_adj name
[  472.880649] [    705]    81   705      308        1    32768       29             0 ubusd
[  472.889131] [    706]     0   706      209        0    32768       12             0 askfirst
[  472.897384] [   1760]   514  1760      302        1    32768       34             0 logd
[  472.905876] [   1808]     0  1808      396        0    32768       31             0 rpcd
[  472.913956] [   2223]     0  2223     1568        1    45056      115             0 hostapd
[  472.922031] [   2224]     0  2224     1539        0    45056       96             0 wpa_supplicant
[  472.930103] [   2281]     0  2281      459        1    32768       53             0 netifd
[  472.939131] [   2336]     0  2336      360        1    28672       30             0 odhcpd
[  472.947376] [   2553]     0  2553     1341        1    40960      144             0 uhttpd
[  472.955616] [   2928]     0  2928      310        0    28672       13             0 udhcpc
[  472.963871] [   2937]     0  2937      248        0    28672       21             0 odhcp6c
[  472.972116] [   3344]     0  3344      259        1    28672       15             0 dropbear
[  472.980369] [   3346]     0  3346      311        1    32768       16             0 ntpd
[  472.989044] [   4194]     0  4194      212        0    28672       14             0 ntpclient
[  472.997118] [   5658]     0  5658     1489        0    40960      101             0 ttyd
[  473.005531] [  10601]     0 10601      311        1    28672       12             0 crond
[  473.013698] [  11749]   453 11749      367        0    32768       36             0 dnsmasq
[  473.021781] [  11796]     0 11796      328        1    28672       35             0 openclash_watch
[  473.029934] [  13146] 65534 13146   178608        0    98304     4784             0 clash
[  473.038960] [  13157]     0 13157      310        0    28672       13             0 sleep
[  473.047290] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/,task=clash,pid=13146,uid=65534
[  473.055490] Out of memory: Killed process 13146 (clash) total-vm:714432kB, anon-rss:0kB, file-rss:0kB, shmem-rss:0kB, UID:65534 pgtables:96kB oom_score_adj:0
[  473.078080] oom_reaper: reaped process 13146 (clash), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB
[  473.078188] hostapd invoked oom-killer: gfp_mask=0x1100cca(GFP_HIGHUSER_MOVABLE), order=0, oom_score_adj=0
[  473.091159] CPU: 1 PID: 2223 Comm: hostapd Not tainted 5.15.34 #0
[  473.100844] Hardware name: Redmi AX6 (DT)
[  473.106997] Call trace:
[  473.110983]  dump_backtrace+0x0/0x160
[  473.113245]  show_stack+0x18/0x30
[  473.117064]  dump_stack_lvl+0x7c/0xa0
[  473.120362]  dump_stack+0x18/0x34
[  473.124008]  dump_header+0x48/0x184
[  473.127307]  oom_kill_process+0x1e0/0x1ec
[  473.130606]  out_of_memory+0x1cc/0x320
[  473.134772]  __alloc_pages_slowpath.constprop.0+0x73c/0xa34
[  473.138421]  __alloc_pages+0x220/0x24c
[  473.143888]  wp_page_copy+0x60/0x870
[  473.147706]  do_wp_page+0xa0/0x430
[  473.151438]  __handle_mm_fault+0x69c/0xa70
[  473.154652]  handle_mm_fault+0x130/0x180
[  473.158730]  do_page_fault+0x14c/0x380
[  473.162809]  do_mem_abort+0x40/0xb0
[  473.166369]  el0_da+0x24/0x60
[  473.169753]  el0t_64_sync_handler+0xe8/0x130
[  473.172880]  el0t_64_sync+0x184/0x188
[  473.177330] Mem-Info:
[  473.180782] active_anon:0 inactive_anon:7 isolated_anon:0
[  473.180782]  active_file:0 inactive_file:103 isolated_file:0
[  473.180782]  unevictable:0 dirty:0 writeback:0
[  473.180782]  slab_reclaimable:1954 slab_unreclaimable:64665
[  473.180782]  mapped:47 shmem:0 pagetables:162 bounce:0
[  473.180782]  kernel_misc_reclaimable:0
[  473.180782]  free:8663 free_pcp:2997 free_cma:0
[  473.195039] Node 0 active_anon:0kB inactive_anon:28kB active_file:0kB inactive_file:412kB unevictable:0kB isolated(anon):0kB isolated(file):0kB mapped:188kB dirty:0kB writeback:0kB shmem:0kB writeback_tmp:0kB kernel_stack:1848kB pagetables:648kB all_unreclaimable? yes
[  473.218991] DMA free:34552kB min:20480kB low:24576kB high:28672kB reserved_highatomic:8192KB active_anon:0kB inactive_anon:28kB active_file:0kB inactive_file:412kB unevictable:0kB writepending:0kB present:524288kB managed:417372kB mlocked:0kB bounce:0kB free_pcp:12356kB local_pcp:4640kB free_cma:0kB
[  473.245495] lowmem_reserve[]: 0 0 0 0
[  473.267697] DMA: 1140*4kB (UMEH) 909*8kB (UMEH) 586*16kB (UMH) 275*32kB (UMH) 49*64kB (UMH) 9*128kB (UMH) 1*256kB (H) 0*512kB 0*1024kB 0*2048kB 0*4096kB = 34552kB
[  473.271252] 131 total pagecache pages
[  473.285671] 0 pages in swap cache
[  473.289396] Swap cache stats: add 9794, delete 9793, find 32/1456
[  473.292677] Free swap  = 202748kB
[  473.298763] Total swap = 207868kB
[  473.302047] 131072 pages RAM
[  473.305343] 0 pages HighMem/MovableOnly
[  473.308319] 26729 pages reserved
[  473.311855] Tasks state (memory values in pages):
[  473.315329] [  pid  ]   uid  tgid total_vm      rss pgtables_bytes swapents oom_score_adj name
[  473.319977] [    705]    81   705      308        1    32768       29             0 ubusd
[  473.328480] [    706]     0   706      209        0    32768       12             0 askfirst
[  473.336714] [   1760]   514  1760      302        1    32768       34             0 logd
[  473.345200] [   1808]     0  1808      396        0    32768       31             0 rpcd
[  473.353311] [   2223]     0  2223     1568        1    45056      115             0 hostapd
[  473.361363] [   2224]     0  2224     1539        0    45056       96             0 wpa_supplicant
[  473.369449] [   2281]     0  2281      459        1    32768       53             0 netifd
[  473.378463] [   2336]     0  2336      360        1    28672       30             0 odhcpd
[  473.386714] [   2553]     0  2553     1341        1    40960      144             0 uhttpd
[  473.394939] [   2928]     0  2928      310        0    28672       13             0 udhcpc
[  473.403210] [   2937]     0  2937      248        0    28672       21             0 odhcp6c
[  473.411464] [   3344]     0  3344      259        1    28672       15             0 dropbear
[  473.419708] [   3346]     0  3346      311        1    32768       16             0 ntpd
[  473.428411] [   4194]     0  4194      212        0    28672       14             0 ntpclient
[  473.436435] [   5658]     0  5658     1489        0    40960      101             0 ttyd
[  473.444867] [  10601]     0 10601      311        1    28672       12             0 crond
[  473.453040] [  11749]   453 11749      367        0    32768       36             0 dnsmasq
[  473.461095] [  11796]     0 11796      328        1    28672       35             0 openclash_watch
[  473.469263] [  13157]     0 13157      310        0    28672       13             0 sleep
[  473.478284] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/,task=uhttpd,pid=2553,uid=0
[  473.486639] Out of memory: Killed process 2553 (uhttpd) total-vm:5364kB, anon-rss:0kB, file-rss:4kB, shmem-rss:0kB, UID:0 pgtables:40kB oom_score_adj:0
[  519.353729] clash invoked oom-killer: gfp_mask=0x1100cca(GFP_HIGHUSER_MOVABLE), order=0, oom_score_adj=0
[  519.353794] CPU: 1 PID: 13647 Comm: clash Not tainted 5.15.34 #0
[  519.362294] Hardware name: Redmi AX6 (DT)
[  519.368280] Call trace:
[  519.372182]  dump_backtrace+0x0/0x160
[  519.374442]  show_stack+0x18/0x30
[  519.378258]  dump_stack_lvl+0x7c/0xa0
[  519.381558]  dump_stack+0x18/0x34
[  519.385203]  dump_header+0x48/0x184
[  519.388502]  oom_kill_process+0x1e0/0x1ec
[  519.391802]  out_of_memory+0x1cc/0x320
[  519.395967]  __alloc_pages_slowpath.constprop.0+0x73c/0xa34
[  519.399617]  __alloc_pages+0x220/0x24c
[  519.405082]  do_swap_page+0x430/0x6b0
[  519.408902]  __handle_mm_fault+0x32c/0xa70
[  519.412635]  handle_mm_fault+0x130/0x180
[  519.416628]  do_page_fault+0x14c/0x380
[  519.420707]  do_translation_fault+0xa0/0xb0
[  519.424266]  do_mem_abort+0x40/0xb0
[  519.428345]  el0_da+0x24/0x60
[  519.431816]  el0t_64_sync_handler+0xe8/0x130
[  519.434944]  el0t_64_sync+0x184/0x188
[  519.439424] Mem-Info:
[  519.442850] active_anon:895 inactive_anon:527 isolated_anon:0
[  519.442850]  active_file:301 inactive_file:326 isolated_file:0
[  519.442850]  unevictable:0 dirty:0 writeback:0
[  519.442850]  slab_reclaimable:1904 slab_unreclaimable:64743
[  519.442850]  mapped:614 shmem:19 pagetables:197 bounce:0
[  519.442850]  kernel_misc_reclaimable:0
[  519.442850]  free:7355 free_pcp:209 free_cma:0
[  519.457811] Node 0 active_anon:3732kB inactive_anon:2184kB active_file:1204kB inactive_file:1304kB unevictable:0kB isolated(anon):0kB isolated(file):0kB mapped:2456kB dirty:0kB writeback:0kB shmem:76kB writeback_tmp:0kB kernel_stack:2032kB pagetables:788kB all_unreclaimable? no
[  519.482367] DMA free:28804kB min:16384kB low:20480kB high:24576kB reserved_highatomic:4096KB active_anon:3836kB inactive_anon:2644kB active_file:1436kB inactive_file:1704kB unevictable:0kB writepending:0kB present:524288kB managed:417372kB mlocked:0kB bounce:0kB free_pcp:1676kB local_pcp:124kB free_cma:0kB
[  519.509362] lowmem_reserve[]: 0 0 0 0
[  519.531592] DMA: 1089*4kB (UMEH) 859*8kB (UMEH) 456*16kB (UMEH) 185*32kB (UMEH) 49*64kB (UMEH) 4*128kB (H) 1*256kB (H) 0*512kB 0*1024kB 0*2048kB 0*4096kB = 28348kB
[  519.535408] 724 total pagecache pages
[  519.549844] 3 pages in swap cache
[  519.553606] Swap cache stats: add 18679, delete 18676, find 145/7173
[  519.556928] Free swap  = 191484kB
[  519.563327] Total swap = 207868kB
[  519.566589] 131072 pages RAM
[  519.569839] 0 pages HighMem/MovableOnly
[  519.572790] 26729 pages reserved
[  519.576371] Tasks state (memory values in pages):
[  519.579824] [  pid  ]   uid  tgid total_vm      rss pgtables_bytes swapents oom_score_adj name
[  519.584457] [    705]    81   705      308      120    32768       28             0 ubusd
[  519.592961] [    706]     0   706      209        0    32768       11             0 askfirst
[  519.601205] [   1760]   514  1760      302        2    32768       36             0 logd
[  519.609710] [   1808]     0  1808      396       89    32768       30             0 rpcd
[  519.617782] [   2223]     0  2223     1595      252    45056       76             0 hostapd
[  519.625840] [   2224]     0  2224     1539        0    45056       93             0 wpa_supplicant
[  519.633949] [   2281]     0  2281      459       23    32768       48             0 netifd
[  519.642962] [   2336]     0  2336      360       79    28672       21             0 odhcpd
[  519.651219] [   2928]     0  2928      310        0    28672       12             0 udhcpc
[  519.659458] [   2937]     0  2937      248        0    28672       20             0 odhcp6c
[  519.667705] [   3344]     0  3344      259        2    28672       14             0 dropbear
[  519.675952] [   3346]     0  3346      311        0    32768       25             0 ntpd
[  519.684610] [   4194]     0  4194      212        0    28672       13             0 ntpclient
[  519.692727] [   5658]     0  5658     1489        0    40960      101             0 ttyd
[  519.701118] [  10601]     0 10601      311       87    28672       11             0 crond
[  519.709295] [  11749]   453 11749      367      177    32768       13             0 dnsmasq
[  519.717348] [  11796]     0 11796      328        1    28672       34             0 openclash_watch
[  519.725496] [  13569]     0 13569     1341      153    45056      116             0 uhttpd
[  519.734549] [  13638] 65534 13638   178608     1384   102400     3040             0 clash
[  519.742874] [  13648]     0 13648      310        0    28672       11             0 sleep
[  519.751142] [  13654]     0 13654      284      200    28672        0             0 luci
[  519.759283] oom-kill:constraint=CONSTRAINT_NONE,nodemask=(null),cpuset=/,mems_allowed=0,global_oom,task_memcg=/,task=clash,pid=13638,uid=65534
[  519.767514] Out of memory: Killed process 13638 (clash) total-vm:714432kB, anon-rss:5532kB, file-rss:4kB, shmem-rss:0kB, UID:65534 pgtables:100kB oom_score_adj:0
[  519.788602] oom_reaper: reaped process 13638 (clash), now anon-rss:0kB, file-rss:0kB, shmem-rss:0kB

**Reply by yaya131** [reply]
集成sfe通用加速。在特定的场景如科学上网，手机连无线科学上网。路由会出现内存骤降至0并死机。具体内核及日志
